packages/core/framework/src/http/express-loader.ts

```diff
@@ -60,33 +60,71 @@ export async function expressLoader({ app }: { app: Express }): Promise<{
     })
   }
 
-  app.set("trust proxy", 1)
+  app.set("trust proxy", true)
```

Comment:
Should we maybe do this in a different PR? It has some security implications and it probably needs a bit more thought, WDYT?

Reply:
We ve discussed it with @sgirones, I will revert it there but the ideal would be to be able to configure it, I ll open a ticket and mentioned oli about it

Reply:
I think even if we do it it might be better if it is a separate PR as it is an important change, security-wise. I leave it up to you guys to decide though :)

Reply:
here is the ticket FRMW-2917

Reply:
I ve reverted it, salvador confirmed that it is not too much of an issue in the short term, but the logs will contains the wrong IP address. I ve open the ticket such that we have a proper discussion around it. Then doing the changes should be quick